aws/eks.tf

```diff
@@ -0,0 +1,71 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "18.26.6"
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.22"
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
+
+  node_security_group_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = null
+  }
+
+  eks_managed_node_group_defaults = {
+    ami_type = "AL2_x86_64"
+
+    attach_cluster_primary_security_group = true
+
+    # Disabling and using externally provided security groups
+    create_security_group = false
+  }
+
+  eks_managed_node_groups = {
+    one = {
```

Comment:
What is the second node group for? (I'm new to AWS and terraform so I might have missed something)

Reply:
Similar question here, a HA-cluster configuration might want two node groups and we should have a HA cluster setup, but the default can be in a single AWS availability zone.

Reply:
This seems to be unresolved

Reply:
I've removed the second node group in a recent commit, but I left the `one = {}` syntax to make it easier to add more node_groups if the user wants to make the cluster HA.

Reply:
What would be the terraform setting to enable HA (assuming this is what you want to support based on recent comment) in this case when leaving `one`?  

It might be worth looking into updating this into[ a dynamic block ](https://developer.hashicorp.com/terraform/language/expressions/dynamic-blocks) and creating node groups based on user input instead of hard-coding just one/two if I'm understanding the purpose correctly?